Classes/Controller/AuthenticationController.php

```diff
@@ -282,10 +282,11 @@ public function startPasswordResetAction($username)
         $this->request->setFormat('txt');
         $message->setBody($this->view->render('passwordResetMail'), 'text/plain');
 
-        $this->request->setFormat('html');
         if ($this->getSettingValue('passwordReset.mail.addHtmlPart')) {
+            $this->request->setFormat('html');
```

Comment:
Won't you render the password reset confirmation page as `txt` then?

Reply:
Won't happen since we `redirect()` afterwards which ensures we start with a fresh request, again defaulting to HTML.

Reply:
Good, then forget what I said :-)